codesystems/CodeSystem-England-PeriodType.xml

```diff
@@ -28,4 +28,8 @@
     <code value="Operational" />
     <display value="Operational" />
   </concept>
+  <concept>
+    <code value="system-effective" />
+    <display value="System Effective" />
+  </concept>
```

Comment:
I'm not sure what the difference is between legal, operational and system effective, so I expect others to be the same. 
Would you be able to add a small sentence on what each mean using the `definition` element https://hl7.org/fhir/R4/codesystem-definitions.html#CodeSystem.concept.definition